sql-plugin/src/main/spark350/scala/org/apache/spark/sql/execution/rapids/shims/FilePartitionShims.scala

```diff
@@ -14,10 +14,10 @@
  * limitations under the License.
  */
 
+
```

Comment:
nit: Just curious why it changes the code formatting in untouched files.

Reply:
I'm assuming our script to update shims did this, but please verify @sririshindra   we could revert this if it was manual

Reply:
That is correct Tom. These changes are from the shimlify script. These are not manual.

Reply:
yes, this is an artifact of upserting the shim comment in shimplify. It's actually sort of normalization of a manual edit. Because it used to be less flexible in parsing and always assumed the line above packages as a fixed position  for the shim descriptor which was a single line CSV